project/backend/collector-config.yaml

```
@@ -0,0 +1,40 @@
+service:
+  pipelines:
+    traces:
+      receivers: [otlp]
+      processors: [batch]
+      exporters: [zipkin, debug]
+    metrics:
+      receivers: [otlp]
+      processors: [batch]
+      exporters: [prometheus, debug]
+    logs:
+      receivers: [ otlp ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)